### PR TITLE
[Fortinet] Separate Fortinet FortiMail into own Integration

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

Separate Fortinet FortiMail into own Integration

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Relates #1936

### CODE REVIEW

**Comment by legoguy1000** [comment]
@andrewkroh ready for testing

**Comment by legoguy1000** [comment]
I found and fixed the issues

File: packages/fortinet_fortimail/data_stream/log/agent/stream/log.yml.hbs [hunk 1]
```diff
@@ -0,0 +1,4300 @@
+paths:
```
Comment (efd6) [inline packages/fortinet_fortimail/data_stream/log/agent/stream/log.yml.hbs:1]: Also in these. Nothing needs to be done.

**Review by efd6 — APPROVED** [review]

**Pull request merged**